cmd/upgrade/accountroles/cmd.go

```diff
@@ -262,8 +262,18 @@ func upgradeAccountRolePolicies(reporter *rprtr.Object, awsClient aws.Client, pr
 		if err != nil {
 			return err
 		}
+
+		err = awsClient.AttachRolePolicy(roleName, policyARN)
+		if err != nil {
+			return err
+		}
+		//Delete if present else continue
+		err = awsClient.DeleteInlineRolePolicies(roleName)
+		if err != nil {
```

Comment:
Should we ignore this error and move on? Or could it cause issues if left in place?

Reply:
@vkareh  We handle it if it is left in place. This error is only if there are any other aws error other than "Not Found" . I have added delete inline policy in the manual mode now as well. Can you PTAL again